src/main/webapp/dashboard.js

```diff
@@ -1,58 +1,72 @@
 google.charts.load('current', {packages: ['corechart', 'line']});
- 
+
 // Render a chart visual on dashboard page for tracking class visits by date
 function drawBasic() {
- 
+
   // Set up the data table to have a class name and visits associated w/ that specific class
   var data = new google.visualization.DataTable();
-  data.addColumn('date', 'Date');
+  data.addColumn('string', 'Date'); // Is this supposed to be string or date?
   data.addColumn('number', 'Visits');
- 
-  let dates = [new Date(2020, 7, 5), new Date(2020, 7, 9), new Date(2020, 7, 15),
-   new Date(2020, 7, 21), new Date(2020, 8, 5)];
-  let numVisits = [12, 1, 8, 9, 4];
+
+  // Organize visit data through visit-by-date servlet
+  fetch(`/visit-date?classCode=` + getParam("classCode"))
+    .then(response => response.json()).then(visits=> {
+    
+    var dates = visits.dates;
+    var numVisits = visits.classVisits;
+
+    var tempDataHolder = []; // To be pushed into datatable after updating
+
+    // Loop through both lists and add info sets for each class 
+    for (var i = 0; i < dates.length; i++) {
+      var temp = [];
+      temp.push(dates[i]);
+      temp.push(numVisits[i]);
+      tempDataHolder.push(temp);
```

Comment:
Do you think in-lining this list will make it more clear `tempDataHolder.push([dates[i], numVisits[i]]);`?